lib/bigcommerce/connection.rb

```diff
@@ -125,6 +125,7 @@ def request_legacy(method, path, options, headers={})
       resource_options = {
           :user => @configuration[:username],
           :password => @configuration[:api_key],
+          :verify_ssl => OpenSSL::SSL::VERIFY_NONE,
```

Comment:
We want to keep the secure default, can you just pass this option in explicitly?